src/test/java/com/synopsys/integration/jenkins/ChangeSetFilterTest.java

```diff
@@ -0,0 +1,90 @@
+package com.synopsys.integration.jenkins;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import com.synopsys.integration.jenkins.extensions.JenkinsIntLogger;
+
+import hudson.scm.ChangeLogSet;
+import hudson.scm.EditType;
+
+public class ChangeSetFilterTest {
+
+    String validPath = "valid_path";
+    String invalidPath = "invalid_path";
+    String dummyPath = "dummy_path";
+
+    EditType editType = new EditType("MockEditTypeName", "MockEditTypeDescription");
+    JenkinsIntLogger MockJenkinsIntLogger = Mockito.mock(JenkinsIntLogger.class);
```

Comment:
Variables should not start with a capital letter. This should be `mockJenkinsIntLogger`

Reply:
We also dont need to create a JenkinsIntLogger. The places you use this take an IntLogger. So we should use `PrintStreamIntLogger(System.out, LogLevel.INFO)`